platform_plugin_forum_email_notifier/utils.py

```diff
@@ -0,0 +1,32 @@
+"""Utilities for the platform_plugin_forum_email_notifier plugin."""
+
+from collections import namedtuple
+
+from openedx.core.djangoapps.django_comment_common.comment_client import settings, utils
+
+Subscriber = namedtuple("Subscriber", ["id"])
+
+
+def get_subscribers(thread_id):
+    """Return a list of user ids subscribed to a thread."""
+    page = 1
+    num_pages = 100
```

Comment:
should we make this configurable? Or at least store them as constants

Reply:
I don't think is needed, those are just initial values that will satisfy the initial while condition

Reply:
so we're always checking just for the first 100 pages?

Reply:
no, this is updated here: 

https://github.com/eduNEXT/platform-plugin-forum-email-notifier/pull/2/files#diff-98cb4bd825d030cff04e0bd6ac2b3c1d8bf7df41f3ee8cfe75d29a31c02d3056R28